Core/Core/People/PeopleListViewController.swift

```diff
@@ -64,14 +68,16 @@ public class PeopleListViewController: UIViewController, ColoredNavViewProtocol,
         view.backgroundColor = .named(.backgroundLightest)
         setupTitleViewInNavbar(title: NSLocalizedString("People", bundle: .core, comment: ""))
 
-        activityIndicatorView.color = Brand.shared.primary
-        emptyResultsLabel.text = NSLocalizedString("No results", bundle: .core, comment: "")
+        emptyMessageLabel.text = NSLocalizedString("We couldn’t find somebody like that.", bundle: .core, comment: "")
+        emptyTitleLabel.text = NSLocalizedString("No Results", bundle: .core, comment: "")
+        errorView.messageLabel.text = NSLocalizedString("There was an error loading people. Pull to refresh to try again.", bundle: .core, comment: "")
+
```

Comment:
I believe the error view retry button needs to be hooked up.
```suggestion
errorView.retryButton.addTarget(self, action: #selector(refresh), for: .primaryActionTriggered)

```